ubuntu_image/common_builder.py

```diff
@@ -211,6 +211,27 @@ def pre_populate_bootfs_contents(self):
                 os.makedirs(target_dir, exist_ok=True)
         self._next.append(self.populate_bootfs_contents)
 
+    @classmethod
+    def _selective_copytree(cls, src, dst):
+        # Selectively copy entries from src to dst directory. Entries already
+        # existing at dst are not overwritten when overwrite is True.
```

Comment:
Is the second part of the sentence still relevant? I don't think we pass any 'overwrite' parameter here, right?